### DISCUSSION TITLE
<HELP Recover deleted items from a view>

### DISCUSSION THREAD

**Discussion by asantos04** [community/community#114489] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was editing some items in a view in my project and i don't know how but all the items i had have been deleted, how can i recover them? 
<img width="1269" alt="Captura de ecrã 2024-03-27 141351" src="https://github.com/community/community/assets/92631447/d65c721d-d809-4b2b-ad5e-1bcdf0ef90d8">

**Comment by benogoulart** [comment]
Hi,

If the items were actually deleted from the project, unfortunately there isn't a built-in **undo** feature or recycle bin to restore them.

Before assuming they're gone, I'd recommend checking a few things:

- Make sure there are no filters applied to the current view.
- Try switching to another project view (such as **Table**) to see if the items are still there.
- Check whether the items were moved to a different status, iteration, or view instead of being deleted.

If the items were permanently removed from the project, you'll likely need to add them back manually. However, if they were linked to existing **GitHub Issues** or **Pull Requests**, those should still exist and can simply be re-added to the project.

Could you clarify whether these were **draft issues** or linked **GitHub Issues/PRs**? Also, is this a **Personal Project** or an **Organization Project**? That will help narrow down the available recovery options.
